x-pack/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -77,6 +79,23 @@ export const TransformManagement: FC = () => {
   // Call useRefreshInterval() after the subscription above is set up.
   useRefreshInterval(setBlockRefresh);
 
+  const unauthorizedTransformsWarning = useMemo(() => {
+    const unauthorizedCnt = transforms.filter((t) => needsReauthorization(t)).length;
+    return unauthorizedCnt > 0 ? (
+      <>
+        <EuiCallOut
+          color="warning"
+          title={i18n.translate('xpack.transform.transformList.unauthorizedTransformsCallout', {
+            defaultMessage:
+              '{unauthorizedCnt, plural, one {A transform was installed but requires more permissions to run. You must have manage_transform cluster privileges to start it.} other {# transforms were installed but require more permissions to run. You must have manage_transform cluster privileges to start them.}}',
```

Comment:
As in my question above, just checking if we should we be calling out specific cluster privileges here?